docs/references/sdk-metrics.mdx

```diff
@@ -486,7 +486,19 @@ internal errors in the SDK.
 
 - Type: Counter
 - Available in: Go, Java
-- Tags: `namespace`, `task_queue`, `nexus_service`, `nexus_operation`
+- Tags: `namespace`, `task_queue`, `nexus_service`, `nexus_operation`, `failure_reason`
+
+Valid values for the `failure_reason` tag:
+
+- `internal_sdk_error`: There was an unexpected internal error within the SDK while handling the Nexus task. Indicates a
+  bug in the SDK.
+- `handler_error_{TYPE}`: The user handler code returned an error with a predefined error specified in the [Nexus
```

Comment:
Looks great! Just put every sentence on one line. 
One small thing (unless there's particular reason for the original): 

```suggestion
`handler_error_{TYPE}`: The user handler code returned a predefined error, as specified in the [Nexus spec](https://github.com/nexus-rpc/api/blob/main/SPEC.md#predefined-handler-errors).
If the handler returns an unexpected error, the TYPE is set to `internal`.